corporate/lib/stripe.py

```diff
@@ -5413,6 +5413,31 @@ def get_plan_renewal_or_end_date(plan: CustomerPlan, event_time: datetime) -> da
     return billing_period_end
 
 
+def maybe_send_fixed_price_plan_renewal_reminder_email(
+    plan: CustomerPlan, billing_session: BillingSession
+) -> None:
+    # We expect to have both an end date and next invoice date
+    # for this CustomerPlan.
+    assert plan.end_date is not None
```

Comment:
This might be better handled as raising a `BillingError`. Previously, a fixed-price plan without an end date wouldn't cause any issues here.

Reply:
Yeah, that seems reasonable.